extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -460,35 +467,24 @@ export class Visibility {
 
     // Remove target that have no listeners.
     if (listeners.length == 0) {
-      this.intersectionObserver_.unobserve(change.target);
+      this.intersectionObserver_.unobserve(target);
```

Comment:
Listener per trigger is clear. But I'm wondering if `.observe` can be called many times like this.